### PR TITLE
Token Refresher

### PR DESCRIPTION
**Pull request by RIO98**
Function "refresh_token_periodically()" added for token update every 59 mins through a daemon thread.

### CODE REVIEW

**Review by MGMCN — APPROVED** [review]
LGTM!

File: app.py [hunk 1]
```diff
@@ -1,100 +1,33 @@
...
-        for item in res["illusts"]:
-            # Title encoding type Unicode
-            returnJson["list"][item["title"]] = packIllustUrl(item["id"])
-        if res["next_url"] is None:
-            break
-        offset = api.parse_qs(res["next_url"])["offset"]
-
-    returnJson["status"] = 1
-    return returnJson
+        json = {"status": 0, "message": "Get error..." + message}
```
Comment (RIO98) [inline app.py:29]: json = {"status": 0, "message": "Get error...%s" % message}

File: services/pixiv.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+
+def packIllustUrl(uid) -> str:
+    return "https://www.pixiv.net/artworks/%s" % uid
+
+
+@singleton
+class Pixiv:
+
+    def __init__(self, username, password, retry=5, interval=3599):
```
Comment (RIO98) [inline services/pixiv.py:27]: def __init__(self, username: str, password: str, retry: int = 5, interval: int = 3599):
        """
        Start the Pixiv service for tg_bot.

        :param username: Given Pixiv username.
        :param password: Given Pixiv password.
        :param retry: Maximum retry times.
        :param interval: Auto-refresh interval for the Pixiv token.
        """

Comment (RIO98) [reply services/pixiv.py:27]: Can can need

**Pull request merged**